src/main.ts

```diff
@@ -69,13 +69,45 @@ app.config.globalProperties.$filters = {
     }
     return featureValue;
   },
+  groupFeatures(featureHierarchy: any) {
+    let features =[] as any;
+    let  featuresValue = '';
+    const featuresList = {} as any;
+    if (featureHierarchy) {
+      featureHierarchy.forEach((feature: any) => {
+        const featureSplit = feature ? feature.split('/') : [];
+        features.push(featureSplit[1]);
+        features = [...new Set(features)];
+        features.forEach((x: any)=>{
+          if(featureHierarchy){
+            featureHierarchy.filter((featureItem: any) => featureItem.includes(featureSplit[0] +"/" +x+"/")).forEach((feature: any) => {
+              const featureSplit = feature ? feature.split('/') : [];
+              const featureValue = featureSplit[2] ? featureSplit[2] : '';
+              featuresValue += ", " + featureValue;
+            })
+            featuresValue = featuresValue.slice(1);
+            featuresValue = featuresValue.trim();
```

Comment:
This can be done in a single line.
```suggestion
            featuresValue = featuresValue.slice(1).trim();
```

Reply:
> This can be done in a single line.

Done.